devel/ansible-podman/playbook.yml

```diff
@@ -0,0 +1,7 @@
+---
+- hosts: localhost
+  become: false
+  vars:
+      ansible_python_interpreter: /usr/bin/python3
```

Comment:
```suggestion
```

This shouldn't be necessary anymore with recent ansible releases.

Reply:
Hum, I probably just cargo culted that from somewhere. How recent is "recent", though? I'm kinda assuming people maaaaay possibly want to use this on e.g. EL 9, though not sure I want to be tied to supporting it.

Reply:
This hasn't been necessary for a while, including on EL 9.